airbyte-cdk/java/airbyte-cdk/dependencies/build.gradle

```diff
@@ -218,3 +221,10 @@ tasks.named('compileJava').configure {
 tasks.named('compileTestJava').configure {
     dependsOn generate
 }
+
+// uses afterEvaluate because at configuration time, the kaptGenerateStubsKotlin task does not exist.
+afterEvaluate {
+    tasks.named('kaptGenerateStubsKotlin').configure {
+        mustRunAfter generateApiServer, generateApiClient, generateJsonSchema2Pojo
+    }
+}
```

Comment:
Ehhh, this is too brittle to my liking. Can you instead add something in `airbyte-cdk/java/airbyte-cdk/build.gradle` in the `allprojects { ... }` block which makes all tasks with names matching `generate.+` depend on all tasks named `kaptGenerateStubsKotlin` and make that one depend on all tasks with names matching `generate`? The `afterEvaluate` won't be necessary since that's where you apply your plugin and register the task.

Reply:
> Ehhh, this is too brittle to my liking. Can you instead add something in `airbyte-cdk/java/airbyte-cdk/build.gradle` in the `allprojects { ... }` block which makes all tasks with names matching `generate.+` depend on all tasks named `kaptGenerateStubsKotlin` and make that one depend on all tasks with names matching `generate`? The `afterEvaluate` won't be necessary since that's where you apply your plugin and register the task.

@postamar. This is really a hack for now.  I want to do another pass and convert to KSP for Kotlin annotation processing which should eliminate the need for this.   We can push it to a common place and do a hack for this, or just get rid of most of this as it pertains mostly to the airbyte-api, which we don't need.

Reply:
@postamar I have switched the PR over to use KSP instead of KAPT.  This removes the hacks called out in your original comment, so you should no longer see additional KAPT configuration spread throughout.